dcat/index.html

```diff
@@ -2495,7 +2495,7 @@ <h4>Property: in series</h4>
                 <!--tr><th class="prop">Inverse property of:</th>
                 <td><a href="#Property:dataset_series_series_member"><code>dcat:seriesMember</code></a></td>
                 </tr-->
-                <tr><th class="prop">Usage note:</th><td> Normally, child datasets in dataset series are represented as <code>dcat:Dataset</code>. The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.</td></tr>
+                <tr><th class="prop">Usage note:</th><td> Child datasets in dataset series SHOULD be represented as <code>dcat:Dataset</code>.<!--The use of <code>dcat:Distribution</code> for typing child datasets is however recognized as a possible alternative, whenever it addresses more effectively the requirements of a given application scenario.--></td></tr>
```

Comment:
This is quite a strong change. It is clearly justified in principle. However, it may not be sustainable in practice. DCAT does not have wide adoption already. Maybe it is more influential than adopted.

Reply:
> This is quite a strong change. 

It might be not that strong. We are using "SHOULD", not "MUST". 

"SHOULD"  means that there may exist valid reasons in particular circumstances to ignore a  particular item, but the full implications must be understood and carefully weighed before choosing a different course.

The change attempts to clarify that using datasets as members in series is the recommended solution. 
Other solutions are possible, but this is the one DCAT 3 considers the most flexible and versatile and the one recommended for the new implementations.

Existing or even future implementations can do otherwise if they have thoroughly considered the consequences. 

We don't want to alienate the existing implementations. Therefore, we explained that current solutions are not incompatible with DCAT 3 during the upgrade to DCAT 3 (see the non-normative section beginning at line 5128).

However,  unless we want to support more than a solution and thus accept interoperability issues intrinsic in the availability of more than one recommended solution, we need to be clear on the recommended approach. 
I am not  100% sure this is the best way to do it.

Do you think a "MAY" instead of "SHOULD" would be more appropriate? Do you have any suggestions on how to convey the message more appropriately?

Reply:
Thanks Riccardo. I'm OK with the proposal. Was just checking, and I note the use of SHOULD.